### PR TITLE
Hierarchies with multiple parents or non-standard root definition

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md [hunk 1]
```diff
@@ -165,15 +165,17 @@ The term `LeveledHierarchy` MUST be applied with a qualifier that can be used to
...
+
+A recursive hierarchy does not need to be as uniform as a leveled hierarchy.
 
-The term `RecursiveHierarchy` can only be applied to entity types, and MUST be applied with a qualifier, which is used to reference the hierarchy in transformations operating on recursive hierarchies, in [grouping with `rolluprecursive`](#Groupingwithrolluprecursive), and in [hierarchy functions](#HierarchyFunctions).
+The recursive hierarchy is described in the model by an annotation of the entity type with the complex term `RecursiveHierarchy` with these properties:
+- The `NodeProperty` contains a path with single-valued segments ending in a primitive property. This path points to the property holding the node identifier of the node in the hierarchy.
+- The `ParentNavigationProperty` allows navigation to the instance or instances representing the parent nodes. It MUST be a nullable single- or collection-valued navigation property path that addresses the entity type annotated with this term. There MUST NOT be any cycles in the traversal of parent links.
```
Comment (HeikoTheissen) [inline odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md:174]: This allows nodes with a null and a non-null parent, that is, roots which are also children. Do we need this?

Comment (ralfhandl) [reply odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md:174]: A collection-valued navigation property is always non-nullable. There can't be null values in the array of entity references.

Maybe rephrase the sentence to "collection-valued or nullable single-valued"?

**Comment by HeikoTheissen** [comment]
Closed in favor of oasis-tcs/odata-specs#1
